public/app/plugins/datasource/prometheus/components/PromExploreQueryEditor.tsx

```diff
@@ -48,7 +48,7 @@ export function PromExploreQueryEditor(props: Props) {
           onKeyDownFunc={onReturnKeyDown}
           value={query.interval || ''}
           hasTooltip={true}
-          tooltipContent={'Needs to be a valid time unit string, for example 5s, 1m, 3h, 1d, 1y'}
+          tooltipContent={'Time units can be used here, e.g.: 5s, 1m, 3h, 1d, 1y (Default if no unit is specified: s)'}
```

Comment:
Can we change the tooltip content sentence so it (visually) looks better? 
![Screenshot from 2020-04-01 11-43-33](https://user-images.githubusercontent.com/6322797/78123224-23f75f80-740e-11ea-9dd8-0f15b55cf19c.png)

Reply:
Do you have a suggestion? (`FormLabel` only accepts string for tooltip, so I can't insert a line break before the parens.)